plugin/evm/types/block_test.go

```diff
@@ -35,9 +35,16 @@ import (
 	"github.com/ava-labs/libevm/common"
 	"github.com/ava-labs/libevm/common/math"
 	"github.com/ava-labs/libevm/crypto"
-	"github.com/ava-labs/libevm/params"
 	"github.com/ava-labs/libevm/rlp"
 	"github.com/ava-labs/subnet-evm/internal/blocktest"
+	"github.com/ava-labs/subnet-evm/params"
+
+	. "github.com/ava-labs/subnet-evm/plugin/evm/types"
```

Comment:
is it needed to import both types packages as `.`? seems a little confusing to me.

Reply:
So a bit like https://github.com/ava-labs/coreth/pull/862/commits/7099e4f36cab7a693834ab3767a8fd127e56502a right? I'm waiting for a reply to see if this is the route we want to take